intercom_flutter_web/example/integration_test/intercom_flutter_web_test.dart

```diff
@@ -84,8 +84,21 @@ void main() {
             name: 'John Doe',
             userId: '1',
             phone: '+37256123456',
-            company: 'Some Company LLC',
-            companyId: '2',
+            company: {
+              'company_id': 'Test id',
+              'name': 'Some Company LLC',
+              "created_at": 123,
+              "plan": "Host",
```

Comment:
we should not use the word "Host" in this MR, because then it violates NDA. Let's use generic "Premium" instead

Reply:
Done @lifedemons !